jobs/Backend/Readme.md

```diff
@@ -1,6 +1,8 @@
 # Mews backend developer task
 
-The task is to implement an [ExchangeRateProvider](Task/ExchangeRateProvider.cs) for Czech National Bank. Find data source on their web. It is up to you to decide which technology (from .NET family) or package to use. Part of the task is to find the source of the exchange rate data and a way how to extract it from there.
+The task is to implement an [ExchangeRateProvider](Task/ExchangeRateProvider.cs) for Czech National Bank. Find data source on their web - part of the task is to find the source of the exchange rate data and a way how to extract it from there.
+
+It is up to you to decide which technology (from .NET family) or package to use. Any code design changes/decisions are also completely up to you.
```

Comment:
It is up to you to decide which technology (from .NET family) or package to use. Any code design changes/decisions to the provided skeleton are also completely up to you.

Reply:
It's a nit but we must be as explicit as we can so there is no confusion at all